src/web/error.rs

```diff
@@ -11,6 +11,8 @@ pub enum Nope {
     ResourceNotFound,
     BuildNotFound,
     CrateNotFound,
+    AuthorNotFound,
```

Comment:
This should be 'owner', not 'author'.
```suggestion
    OwnerNotFound,
```

Reply:
Renamed.